semgrep/tests/qa/test_public_repos.py

```diff
@@ -109,3 +114,19 @@ def test_semgrep_on_repo(run_semgrep_in_tmp, repo_url):
         assert output["errors"] == []
         assert len(output["results"]) == 1
         assert output["results"][0]["path"] == str(sentinel_path)
+
+    output = subprocess.check_output(
+        [
+            "python3",
+            "-m",
+            "semgrep",
+            "--config=rules/regex-sentinel.yaml",
```

Comment:
```suggestion
            "--config",
            Path(TESTS_PATH / "qa" / "rules" / "regex-sentinel.yaml").resolve(),
```

wouldn't this work? then we wouldn't need to mess with tmp paths above.

Reply:
I think you need to wrap with `str(...)`

Reply:
nope, subprocess functions accept `Path`

Reply:
Ah, now I see that most of the tmp_path lines would need to stay anyway cause we got rid of the `run_semgrep_in_tmp` fixture which would normally set it up.